src/apigateway/router.js

```diff
@@ -48,16 +50,24 @@ class Router {
         const method = this._getExistingMethod(endpoint);
         const request = new RequestClient(this._event);
         const response = new ResponseClient();
-        const validator = new RequestValidator(request, response, this._schemaPath);
+        const schema = new Schema(this._schemaPath);
+        const requestValidator = new RequestValidator(request, response, schema);
+        const responseValidator = new ResponseValidator(request, response, schema);
+
         if (!response.hasErrors && endpoint.requirements && endpoint.requirements[method]) {
-            await validator.requestIsValid(endpoint.requirements[method]);
+            await requestValidator.isValid(endpoint.requirements[method]);
         }
         if (!response.hasErrors && this._beforeAll && typeof this._beforeAll === 'function') {
             await this._beforeAll(request, response, endpoint.requirements);
         }
         if (!response.hasErrors) {
             await endpoint[method](request, response);
         }
+
+        if (!response.hasErrors && endpoint.requirements && endpoint.requirements[method]) {
+            await responseValidator.isValid(endpoint.requirements[method]);
+        }
```

Comment:
This is a big change and need to configurable per endpoint.

Think of all the use-cases and other library users who use this package and don't want or need response validation.

This needs to be under a `responseBody` param in the requirements export and this needs to check if it exists before attempting to validate

Reply:
Fixed